### ISSUE TITLE
Building a new website for Jenkins X

### ISSUE THREAD

**Issue by rajatgupta24** [jenkins-x/jx-docs#3627]
# Enhancement proposal for new Jenkins X website

Greetings to all.

We are already beginning the process of redesigning the Jenkins X website after talking with the maintainers during office hours.
We want to work on the landing page for the first step.

For it, we're making a Figma design (not GA).

## Problem Statement

The present website is functional, but it doesn't appear to be consistently updated. It provides a lot of information but finding it can be challenging because of how poorly it is organized.
Things we'll like to change:

- The Illustrations are old
- Old Twitter posts 

## Proposed solution
Our process begins with creating a design, followed by

- Adding an introductory video 
- Adding a new CDF Banner
- Also add a few more links to some important resources.

## To contribute:
If you are interested in contributing, please comment
```
I want to <part-in-which-you-are-interested>
```
## Approvers: 

@ankitm123 
@babadofar 
@msvticket 
@tomhobson 

## Reviewers:

@__pro__coder__
@rajatgupta24

**Comment by subhamgcon** [comment]
I want to work on the documentation part. Is this issue still open? @rajatgupta24

**Comment by rajatgupta24** [comment]
> I want to work on the documentation part. Is this issue still open? @rajatgupta24

Yeah, we are still working on the documentation part, you can join too.
